docs/start/getting_started.md

```diff
@@ -0,0 +1,52 @@
+# Getting started
+This guide will help you with the installation of flowchem.
+
+## Overview
+In a typical scenario, flowchem:
+1. reads a configuration file listing the devices to be controlled and their settings;
+2. creates connections with each device and ensures a reproducible state at start-up;
+3. provides access to the capabilities of each device (such as pumping, heating etc...) via a web interface.
+
+:::{figure-md} flowchem-architecture
+<img src="architecture_v1.svg" alt="Flowchem software architecture (devices/config/server)" class="bg-primary mb-1" width="100%">
+
+**Figure 1** Schematic representation of flowchem software architecture.
+A heterogeneous collection of devices is physically connected to a control PC.
+The configuration file in TOML format specifies the connection parameters for each device.
+After running flowchem with that configuration, a web server is started to control each device via a single API.
+:::
+
+### Interoperability
+Since flowchem leverages web technologies, flowchem devices can be controlled directly with a web browser or by clients
+written in different languages and from almost any operative system, including Android and iOS.
```

Comment:
i don't know cellphone application of Android and iOS will be too broad or not (or we need to provide some evidence

Reply:
We gonna talk about it later